breathecode/authenticate/serializers.py

```diff
@@ -30,10 +30,15 @@ class TokenSmallSerializer(serpy.Serializer):
     user = UserTinySerializer()
     key = serpy.Field()
     reset_password_url = serpy.MethodField()
+    reset_github_url = serpy.MethodField()
 
     def get_reset_password_url(self, obj):
         return os.getenv('API_URL') + '/v1/auth/password/' + str(obj.key)
 
+    def get_reset_github_url(self, obj):
+        return os.getenv('API_URL') + '/v1/auth/github/' + str(
+            obj.key) + '?url=https://learn.breatheco.de/login'
```

Comment:
This url should be great implement one custom callback url (`https://learn.breatheco.de/login`)

Reply:
otra vez no entiendo, el url ya estaba definido en el issue asi, def get_reset_github_url(self, obj):
        return os.getenv('API_URL') + '/v1/auth/github/' + str(obj.key) + "?url=https://learn.breatheco.de/login"

Reply:
Si, pero puede ser mejorado, este enlace solo funcionaria para ser usado en https://learn.breatheco.de/login en casos en el que se le reseteara el enlace a alguien del staff, no tendria sentido que usara la plataforma del estudiante para este proposito